doc/community-build/Single-Node-K8s-Cortx-Deployment.md

```diff
@@ -0,0 +1,416 @@
+    glusterfs:
+      size: 5Gi
+  storage:
+    cvg1:
+      name: cvg-01
+      type: ios
+      devices:
+        metadata:
+          device: /dev/sdc
+          size: 5Gi
+        data:
+          d1:
+            device: /dev/sdd
+            size: 5Gi
+          d2:
+            device: /dev/sde
+            size: 5Gi
+    cvg2:
+      name: cvg-02
+      type: ios
+      devices:
+        metadata:
+          device: /dev/sdf
+          size: 5Gi
+        data:
+          d1:
+            device: /dev/sdg
+            size: 5Gi
+          d2:
+            device: /dev/sdh
+            size: 5Gi
+  nodes:
+    node0:
+        name: cortx-testbench
+```
+
+**Run prerequisites script:**
+The prerequisites script configs required packages and changes in the system. Run prerequisite script on the node. "<disk>" is a required input to run this script. This disk should NOT be any of the devices listed in "solution.storage.cvg*" in the "solution.yaml" file. The prerequisites script can also take optional parameter of `<solution-file>`. Please make sure to use the same solution file for prerequisites, deploy and destroy scripts.
+```
+#Example:
+./prereq-deploy-cortx-cloud.sh /dev/sdb
+
+./prereq-deploy-cortx-cloud.sh /dev/sdb [<solution-file>]
+```
+
+**Deploy Cortx cluster:**
+To deploy Cortx cluster framework run any one of the following command.  The deploy script can also take optional parameter of `<solution-file>`.
+```
+#Example:
+./deploy-cortx-cloud.sh
+
+./deploy-cortx-cloud.sh [<solution-file>]
+```
+**Verify Cortx cluster status:**
+After deploy script completes it's implementation all the pods should be up and running. Verify with the following commands:
+```
+kubectl get pods -A
+
+# In the following replace cortx-data-pod-XXXX with actual pod name running in the kubernetes cluster.  
+kubectl exec -it cortx-data-pod-XXXX -c cortx-motr-hax -- hctl  
+status
+```
+
+
+**Destroy Cortx cluster:**
+To destroy Cortx cluster framework run any one of the following command. There is an additional force option if the which can be used in case of any error to remove Cortx cluster components.  The destroy script can also take optional parameter of `<solution-file>`.
+```
+#Example:
+./destroy-cortx-cloud.sh
+
+./deploy-cortx-cloud.sh [<solution-file>]
+
+./destroy-cortx-cloud.sh [<solution-file>] [--force|-f]
+```
+
+# S3 setup:
+
+Now the Cortx cluster is deployment is complete, all pods are in running state and data pods are running, we can setup and use the S3 storage by following the steps below (all the commands are run as `root`  user)-:
+
+## Installation
+
+Install the following packages which will be required in S3 IO operations:
+
+**Install cortx-s3iamcli:**
+```
+yum install -y <link address for cortx-s3iamcli-devel.rpm>  --nogpgcheck
+
+#Example:
+yum install -y http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.9.2009/last_successful/cortx-s3iamcli-devel-2.0.0-248_gite0c1f347.noarch.rpm --nogpgcheck
```

Comment:
Please remove the `cortx-storage.colo.seagate.com` link. This will not be accessible to the community. We need to provide steps to build `cortx-s3iamcli-devel` package. Refer - https://github.com/Seagate/cortx/blob/main/doc/community-build/Generate-Cortx-Build-Stack.md

Reply:
Done